src/Search.js

```diff
@@ -1,23 +1,40 @@
-import React from "react";
+import React, { useState } from "react";
+import SearchButton from "./SearchButton";
+
+const Search = props => {
+  const [searchInput, setSearchInput] = useState("");
```

Comment:
Good job here setting up the state and setting it whenever the user types a new character 👍